### DISCUSSION TITLE
Tracks ending in wrong artists titles

### DISCUSSION THREAD

**Discussion by notevenonce** [tidal-music/discussions#75] [Report bug]
Good day fellows

Not sure this is the way to do this, but basically I would like to make it a habit of signaling tracks that are being uploaded under wrong playlists due to the same name of artists. I will give you an example with the newest Poppy single New Way Out.

 1 I was not notified of this new release since the track ended in the wrong artists
2 There are many artists with the same name which end up stealing tracks from the original artists

Another bug I identified is a song from Korn that actually plays I Miss You by Blink 182. Couldn't find it now, but I will once it ends up in my queue again.

Where should I be signaling these things? 

Thanks.

**Comment by stefanmoro** [comment]
Hi @notevenonce ,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to our existing product offering, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
